python/docs/source/getting_started/install.rst

```diff
@@ -165,16 +168,92 @@ To install PySpark from source, refer to |building_spark|_.
+========================== ========================= ======================================================
+
+Pandas API on Spark
+^^^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pyspark[pandas_on_spark]"``.
+
+========================== ========================= ======================================================
+Package                    Supported version         Note
+========================== ========================= ======================================================
+`pandas`                   >=1.4.4                   Required for utilizing the Pandas API features in Spark.
+`pyarrow`                  >=10.0.0                  Ensures efficient data handling and performance in Pandas operations.
+`numpy`                    >=1.21                    Facilitates complex numerical operations within Spark.
+========================== ========================= ======================================================
+
+Note: Run ``pip install "pyspark[pandas_on_spark] plotly"`` if you want to use visualization features.
+
+ML
+^^
+
+Installable with ``pip install "pyspark[ml]"``.
+
 ========================== ========================= ======================================================================================
-Package                    Supported version Note
+Package                    Supported version         Note
 ========================== ========================= ======================================================================================
-`py4j`                     >=0.10.9.7                Required
-`pandas`                   >=1.4.4                   Required for pandas API on Spark and Spark Connect; Optional for Spark SQL
-`pyarrow`                  >=10.0.0                  Required for pandas API on Spark and Spark Connect; Optional for Spark SQL
-`numpy`                    >=1.21                    Required for pandas API on Spark and MLLib DataFrame-based API; Optional for Spark SQL
-`grpcio`                   >=1.62.0                  Required for Spark Connect
-`grpcio-status`            >=1.62.0                  Required for Spark Connect
-`googleapis-common-protos` >=1.56.4                  Required for Spark Connect
+`numpy`                    >=1.21                    Supports advanced data manipulation and algorithm implementation in ML.
 ========================== ========================= ======================================================================================
 
-Note that PySpark requires Java 17 or later with ``JAVA_HOME`` properly set and refer to |downloading|_.
+MLlib
+^^^^^
+
+Installable with ``pip install "pyspark[mllib]"``.
+
+========================== ========================= ======================================================================================
+Package                    Supported version         Note
+========================== ========================= ======================================================================================
```

Comment:
Let's match the number of `=` to the text

Reply:
matched!